Controller/ResettingController.php

```diff
@@ -130,20 +132,25 @@ public function resetAction(Request $request, $token)
                 $response = new RedirectResponse($url);
             }
 
-            $dispatcher->dispatch(FOSUserEvents::RESETTING_RESET_COMPLETED, new FilterUserResponseEvent($user, $request, $response));
+            $dispatcher->dispatch(
+                FOSUserEvents::RESETTING_RESET_COMPLETED,
+                new FilterUserResponseEvent($user, $request, $response)
+            );
 
             return $response;
         }
 
-        return $this->render('FOSUserBundle:Resetting:reset.html.twig', array(
-            'token' => $token,
-            'form' => $form->createView(),
-        ));
+        return $this->render(
+            'FOSUserBundle:Resetting:reset.html.twig',
+            array(
+                'token' => $token,
+                'form' => $form->createView(),
+            )
+        );
     }
 
     /**
      * Get the truncated email displayed when requesting the resetting.
-     *
```

Comment:
the empty line between the short description and the long description must be kept. It is needed in phpdoc